app/models/user_option.rb

```diff
@@ -14,6 +14,7 @@ class UserOption < ActiveRecord::Base
   scope :human_users, -> { where('user_id > 0') }
 
   enum default_calendar: { none_selected: 0, ics: 1, google: 2 }, _scopes: false
+  enum sidebar_list_destination: { default: 0, unread_new: 1 }, _scopes: false, _default: SiteSetting.default_sidebar_list_destination
```

Comment:
I might be wrong here but referring to site settings when loading a class will break our ability to start the application when access to the database is limited.

Reply:
Good point, I can do something like `SiteSetting.default_sidebar_list_destination rescue "default"` but it sounds even more hacky. 

In that case, I feel that it would be better to drop `_default` and keep default on serializer level.

Reply:
Can we set the default on user create instead? That way, we don’t allow the column to be blank

Reply:
Problem is that if one day we decide to switch site setting from "default" to "unread_new", then we will not have an information if UserOption was set by user or not. 

Maybe we need 3rd enum like `{0: "not_selected", 1: "default", 2: "unread_new"}`. In that case we can default column to 0 and on serializer level still inherit setting from SiteSetting.

Does it sound good to you?

Reply:
O that sounds better actually